src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,79 @@
+package duke;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+
+import java.io.IOException;
+import java.io.File;
+import java.io.FileWriter;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public class Storage {
+
+    public static final String savedFile = "data/storage.txt";
+    public static final String savedFolder = "data";
+    public Task task;
+
+    public Storage() {}
+    public Storage(Task task) {
+        this.task = task;
+    }
+
+    public static List<Task> readFile() throws IOException {
+        List<Task> todoList = new ArrayList<>();
+        Path path = Paths.get(savedFile);
+        List<String> list = new ArrayList<>(Files.readAllLines(path));
+        for (String task : list) {
+            Task current = null;
```

Comment:
Will it be clearer if 'current' is named 'currentTask' instead?

Reply:
Agree with this comment, though as this is just a temporary variable, it is not too big an issue.